### DISCUSSION TITLE
blog/react-hydration

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#389] [General]
# blog/react-hydration

Learn how React hydration works and how to troubleshoot the hydration error

https://theodorusclarence.com/blog/react-hydration

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Comment by briyk** [comment]
i love ur articles bro , please keep going

**Reply by theodorusclarence** [reply]
Thanks!

**Comment by ezechuka** [comment]
Hello Clarence, I just discovered your personal site from your reddit post and so far I've learnt alot since yday. I have a question though, I think useEffect will cause page flashing if hours is > 12. Is it possible to prevent it?
